content/scholar/GIS4-2019/02-network.md

```diff
@@ -78,7 +78,7 @@ ip address del dev eno2 192.168.0.42/24
 Network cards and IPs
 ===
 
-### List neighbors
+### List neighbours
```

Comment:
Correct in american english, but I still changed it to better batch the command argument